client/src/sockets/helpers/index.js

```diff
@@ -1,10 +1,125 @@
 import { CohortEvents, ListEvents } from 'sockets/enums';
 import history from 'common/utils/history';
+import { createNotificationWithTimeout } from 'modules/notification/model/actions';
+import { MessageType as NotificationType } from 'common/constants/enums';
 
 export const listEventsController = (event, data, { dispatch, getState }) => {
   const { currentUser } = getState();
 
   switch (event) {
+    case ListEvents.ADD_OWNER_ROLE_SUCCESS: {
+      const {
+        notificationData: { listName, performer, userName },
+        userId,
+        ...rest
+      } = data;
+      const notificationId = 'list.actions.current-user-add-to-owners';
+      const notificationData = {
```

Comment:
why are you preparing data if it might not be used? Put it inside the `if` statement